structuralcodes/geometry/profiles/_w.py

```diff
@@ -0,0 +1,1716 @@
+"""IPE profiles."""
```

Comment:
Please update the docstring with the correct profile name.
```suggestion
"""W profiles."""
```